### PR TITLE
[fix #3316] Add alternate/square product logo.

### LINKED ISSUE #3316: Add ability to attach a smaller product logo for product pages
[The PR that replaced the product logos with the "?" SVG](https://github.com/mozilla/kitsune/pull/3255) [appears to have done so on purpose](https://github.com/mozilla/kitsune/pull/3255/files#diff-9fb8e903b164ade55bd42033512b9b29R21).

Are there mockups showing what should be used in place of the "?" SVG?

[original comment](https://github.com/mozilla/kitsune/pull/3312#issuecomment-421391396)

### PR DESCRIPTION
**Pull request by jpetto**
This PR is step 1 of 2. 

Step 2 will be updating the UI to match the designs (offsetting the alternate/square logo to the left of the content in a couple spots).

### CODE REVIEW

**Comment by pmac** [comment]
Oh. Nm on the help text. I see you did do that. Maybe the admin is automatically done then?

**Comment by pmac** [comment]
Okay. I've run this locally and the admin is good! So never mind on my whole comment above. The only suggestion I have is to further improve the `help_text` to better explain the purpose of the two image fields. Something like `Image for the product list pages` and `Image for the product sub-pages` respectively. Doesn't have to be that, but something that explains what they're for to go along with the already helpful size recommendation.

**Comment by pmac** [comment]
Circle is complaining about some flake8 stuff.

**Comment by pmac** [comment]
Also I can't find the `product_placeholder_alternate.png` file that should be the image when no image is in the db.

**Comment by jpetto** [comment]
Updated the help text (good suggestion!) and updated `.gitignore` so the added image was detected.

File: .gitignore [hunk 1]
```diff
@@ -34,6 +34,5 @@ k8s/regions/*/kubectl
 k8s/regions/*/*-secrets.yaml
 *.db
 *.sqlite3
-static
```
[the remark this answers is not in the document]
Comment (pmac) [reply .gitignore]: Aaannnnddddd you should entirely ignore me. I now see that the correct `/static` is above this and this line should indeed be removed :) Nice work 💯

**Pull request merged**